server/test/controllers/book.spec.js

```diff
@@ -0,0 +1,192 @@
+import supertest from 'supertest';
+import { assert } from 'chai';
+import app from '../../app';
+import mock from '../mock/mock';
+
+
+const server = supertest.agent(app);
+let jwtToken;
+
+describe('Book', () => {
+  before((done) => {
+    server
+      .post('/api/v1/users/signin')
+      .send(mock.adminUser)
+      .end((err, res) => {
+        jwtToken = res.body.token;
+        done();
+      });
+  });
+  describe('getAllBooks', () => {
+    it('should be allow users to view all books', (done) => {
+      server
+        .get('/api/v1/books')
+        .expect(200)
+        .end((err, res) => {
+          assert.equal(res.status, 200);
```

Comment:
For your tests, you should not only test your status codes. To make this more comprehensive, test the response payload also.

Reply:
Noted. Thank you